### ISSUE TITLE
Info Icons on Medication Suggestor Screens are Not Keyboard Accessible

### ISSUE THREAD

**Issue by Scaranman** [CodeForPhilly/balancer-main#488]
Date: 3/31/26
Pages: Medication Suggestion Patient Details and Summary Page

Link: [https://balancerproject.org/](https://balancerproject.org/)

Device(s): Desktop/Laptop

Phase of Balancer work: 2.0 Part of Content Audit stage after MVP

Description of Problem: 
When using keyboard navigation, the info icons next to fields are inaccessible.

Expected Behavior:
Users should be able to focus the info icons in the medication suggester using keyboard navigation

Browser(s): Chrome, Firefox
